includes/functions.php

```diff
@@ -1579,6 +1587,17 @@ function is_actor( $data ) {
 	 */
 	$types = apply_filters( 'activitypub_actor_types', Actor::TYPES );
 
+	return _is_type_of( $data, $types );
+}
+
+/**
+ * Private helper to check if $data is of a given type set.
+ *
+ * @param array|object|string $data  The data to check.
+ * @param array               $types The types to check against.
+ * @return boolean True if $data is of one of the types, false otherwise.
```

Comment:
```suggestion
 * @param array               $types The types to check against.
 *
 * @return boolean True if $data is of one of the types, false otherwise.
```